packages/flutter/lib/src/widgets/scroll_simulation.dart

```diff
@@ -66,6 +72,10 @@ class BouncingScrollSimulation extends Simulation {
     assert(_springTime != null);
   }
 
+  /// The maximum velocity that can be transfered from the inertia of a ballistic
+  /// scroll into overscroll on iOS.
```

Comment:
I wouldn't say "on iOS" here. It's the maximum velocity that can be transfered from the inertia of a ballistic scroll into overscroll in general, when this simulation is used.